### PR TITLE
Dockerfile fixes

### PR DESCRIPTION
**Pull request by rojer**

### CODE REVIEW

File: Build/Docker/Dockerfile [hunk 1]
```diff
@@ -6,17 +6,13 @@ RUN echo ========== Install dependencies ========== \
...
     perl \
     unzip \
     wget \
-    libasound2-dev \
-    libavahi-compat-libdnssd-dev \
-    libfaac-dev \
-    libopus-dev \
-    libsqlite3-dev \
-    libssl-dev \
-    openssh-server \
```
Comment (aajain-com) [inline Build/Docker/Dockerfile]: Why are we removing `openssh-server` from the list. It does get automatically installed as part of other packages as dependency but good to explicitly install it.

Comment (rojer) [reply Build/Docker/Dockerfile]: it's not required for building, so why have it installed? or is there another use for it that i'm missing?

Comment (aajain-com) [reply Build/Docker/Dockerfile]: `make shell` lets you ssh into the docker container, which requires openssh to be installed.

Comment (rojer) [reply Build/Docker/Dockerfile]: fair enough. never used it myself, so wasn't even aware of it.

Comment (rojer) [reply Build/Docker/Dockerfile]: added back, verified that `make shell` works. PTAL.

**Comment by rojer** [comment]
thank you! while i have your attention, can you perhaps take a look at #12 as well?

**Pull request merged**